### PR TITLE
Remember current tab

### PR DESCRIPTION
**Pull request by bernhardh**
With this solution, you can for example reload (`CTRL+R`) the edit and detail views or you can submit the form page and you will get the previous openend tab back again. And you can also deeplink specific tabs.

ezgif com-video-to-gif

### CODE REVIEW

**Comment by bernhardh** [comment]
@bajramemini: Any chance to get merged?

**Comment by skynet01** [comment]
I support this merge request :)

**Comment by 949mac** [comment]
+1 for merge request

**Comment by bernhardh** [comment]
Still not merged?

**Comment by 949mac** [comment]
> Still not merged?

😧 I know, right!?

**Comment by RVxLab** [comment]
Sorry it has taken this long to address this. I'll take a look at it and come back to you.

**Comment by keironwaites** [comment]
+1

Deep linking would be fantastic.

File: resources/js/components/DetailTabs.vue [hunk 1]
```diff
@@ -119,6 +114,7 @@ export default {
     handleTabClick(tab, event) {
       tab.init = true;
       this.activeTab = tab.name;
+      this.$router.push({ query: { tab: tab.name }});
```
Comment (RVxLab) [inline resources/js/components/DetailTabs.vue:117]: Can you also add a replace here like in FormTabs.vue?

**Review by RVxLab — CHANGES_REQUESTED** [review]
If you could do the 1 small change request and rebase this with the current master branch then this can be merged. This is a fantastic change that I'd love to have in the package. :+1:

File: resources/js/components/DetailTabs.vue (hunk 1 above)
Comment (bernhardh) [reply resources/js/components/DetailTabs.vue]: I hope I understand it right, but I have done the change (and the merge of `master` into my `remember-current-tab branch`)

**Review by RVxLab — APPROVED** [review]

**Comment by RVxLab** [comment]
Perfect, thank you very much!

**Comment by keironwaites** [comment]
Sorry to be impatient but what is the release time schedule for a change like this? :)

**Comment by RVxLab** [comment]
@keironwaites I want to add at least 1 more bugfix, see #23. I'm aiming this weekend.

**Pull request merged**